server/package.json

```diff
@@ -83,6 +84,7 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^3.4.0",
     "jest": "^27.0.6",
+    "pino-pretty": "^6.0.0",
```

Comment:
Should we add `https://www.npmjs.com/package/@types/pino` too?

Reply:
It's already added as a part of [dependency](https://github.com/ToolJet/ToolJet/pull/600/files#diff-be330df323f72d45d073e311befe92984fad9846bb7e982261d702d805c14053R16260) from `nestjs-pino`